xactserver/src/manager.rs

```diff
@@ -196,13 +195,14 @@ impl XactStateManager {
 
                 // TODO: This is sending one-by-one to all other peers, but we want to send
                 //  to just relevant peers, in parallel.
-                for i in 0..self.peers.size() {
+                for i in 1..self.peers.size() {
```

Comment:
Why? Shouldn't we start from 0 always?

Reply:
We are checking node_id != self.node_id in line 200?

Reply:
The node id is the same as the region id and I have reserved the id 0 for the "global region" (which is for data that is not specific to any region), so the node id must start from 1.